core/src/main/resources/hudson/ProxyConfiguration/help-noProxyHost.html

```diff
@@ -1,4 +1,4 @@
 <div>
     Specify host name patterns that shouldn't go through the proxy, one host per line.
-    "*" is the wild card host name (such as "*.cloudbees.com" or "www*.jenkins-ci.org")
+    "*" is the wild card host name (such as "*.jenkins.io" or "www*.jenkins-ci.org")
```

Comment:
Uh oh, thanks for cleaning it up.
Maybe the second example with `www*.jenkins-ci.org` should be somehow replaced as well